packages/nextjs-drupal-webform/src/drupal.ts

```diff
@@ -0,0 +1,16 @@
+import { DrupalClient } from 'next-drupal';
```

Comment:
I don't think we should initialize new Drupal Client in here. I think it needs to be something that is provided by the integrator from the Next.js application.